auth/realm/ldap/src/main/java/org/wildfly/security/auth/realm/ldap/LdapSecurityRealm.java

```diff
@@ -1263,7 +1263,11 @@ public void objectChanged(NamingEvent evt) {
 
```

Comment:
I did not log methods objectChanged, objectAdded, objectRemoved, objectRenamed as it could produce too many trace messages if users would be modifying their databases